where-to-travel/src/main/webapp/signup.js

```diff
@@ -0,0 +1,61 @@
+// Get elements for authentication
+const emailSignUp = document.getElementById('emailSignUp');
+const passwordSignUp = document.getElementById('passwordSignUp');
+const passwordConfirmation = document.getElementById('passwordConfirmation');
+const btnSignUp = document.getElementById('signUp');
+
+// Add signup event
+btnSignUp.addEventListener('click', e => {
+ 
+  const email = emailSignUp.value;
+  const pass = passwordSignUp.value;
+  const passConfirmation = passwordConfirmation.value;
+  const auth = firebase.auth();
+
+  if (validate()==false) {
+    return;
```

Comment:
Should we return an alert/error for this case?

Reply:
Notifying the user is done in the validate function so there is an alert for that case.

Reply:
It might be worth adding a comment about this.

Reply:
ok done